<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataOutputWidget</class>
 <widget class="QWidget" name="DataOutputWidget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="Commwidget" native="true">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>0</y>
     <width>811</width>
     <height>761</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>9</y>
     <width>117</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="IOpushButton">
      <property name="minimumSize">
       <size>
        <width>115</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>115</width>
        <height>41</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>IO配置</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ExternalpushButton">
      <property name="minimumSize">
       <size>
        <width>115</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>115</width>
        <height>41</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>外部通信</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="InsidepushButton">
      <property name="minimumSize">
       <size>
        <width>115</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>115</width>
        <height>41</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>多机互联</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="PushButton_Data_Input">
      <property name="minimumSize">
       <size>
        <width>115</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>115</width>
        <height>41</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>G_Data</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="PushButton_HeartBeat">
      <property name="minimumSize">
       <size>
        <width>115</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>115</width>
        <height>41</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>心跳监控</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
